central/graphql/resolvers/resolver.go

```diff
@@ -142,7 +144,8 @@ func New() *Resolver {
 		NamespaceDataStore:          namespaceDataStore.Singleton(),
 		NetworkPoliciesStore:        npDS.Singleton(),
 		NetworkFlowDataStore:        nfDS.Singleton(),
-		NodeGlobalDataStore:         nodeDataStore.Singleton(),
+		NodeGlobalDataStore:         nodeGlobalDataStore.Singleton(),
```

Comment:
is this needed anymore?

Reply:
Still some instances, which will be addressed in follow ups. This PR focuses on global search functionality.